Userland/Libraries/LibCore/SOCKSProxyClient.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022, Ali Mohammad Pur <mpfard@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/OwnPtr.h>
+#include <LibCore/Stream.h>
+
+namespace Core {
+class SocksProxyClient final : public Stream::Socket {
```

Comment:
I noticed this file's name is capitalized differently, I don't think you meant to do that. :^)

Reply:
yeah I'll rename the class to "SOCKS", that's the official name anyway